app/controllers/allocations_controller.rb

```diff
@@ -2,40 +2,15 @@ class AllocationsController < ApplicationController
   before_action :authenticate_user
 
   def index
-    allocated
-    render 'allocated'
-  end
-
-  def allocated
-    breadcrumb 'Allocated', :allocations_allocated_path
-
     response = OffenderService.new.get_offenders_for_prison(
       caseload,
       page_number: page_number
     )
-    @prisoners = response.data
-    @page_data = response.meta
-  end
 
-  def awaiting
-    breadcrumb 'Awaiting allocation', :allocations_awaiting_path
+    @offenders_allocated = response.data
```

Comment:
Perhaps reduce to one variable? `@offenders`

Reply:
The point of this PR is so that when we submit the next (grouped offenders according to allocated, unallocated, missing_info) we can just return these three objects without having to re-do the HTML.